### DISCUSSION TITLE
VS Code crashes when opening remote repo

### DISCUSSION THREAD

**Discussion by adamyonk** [blinksh/blink#2073] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Blink: v18.0.1.940 (App Store). Oct  4 2024
iOS 18.2 beta
### Describe the bug

See included video, trying to open a remote repo crashes VS Code and renders the blink tab unusable.

I’m not sure if this belongs here or in the remote repositories VS Code extension, but I thought I’d start here. 

https://github.com/user-attachments/assets/93f44652-ee0f-4335-b122-d55e94a1405a

**Comment by adamyonk** [comment]
It’s kind of hard to see in the video with the black text on dark background, but the selected text at the end says `The request is blocked`, followed by a timestamp and some sort of ID.

**Comment by carloscabanero** [comment]
Blink Code is just a thin layer over the browser to connect to our extension, our SW keyboard and little more. So most of what you see falls on the Vscode.dev side of things. They may have broken something in the reload. One thing we can try while they fix things. Have you tried starting the remote environment in the browser and then getting the URL or doing "Open in Desktop" so it will open in Blink? That will skip the reload part and put you directly in the proper session.

**Reply by adamyonk** [reply]
Ah, it does happen in a browser as well. 
![IMG_3202](https://github.com/user-attachments/assets/f2a592df-27bd-4682-8880-aad66bf24f15)

Should I take this directly to the vscode.dev repo? Sorry, I’m not very familiar with this ecosystem!

**Reply by carloscabanero** [reply]
Yep, upstream would be the way to go.

Have you tried my workaround though?

**Reply by adamyonk** [reply]
I did, and it didn’t work either. Might have something to do with a private repo? Started an issue here: https://github.com/microsoft/vscode-remote-repositories-github/issues/495
